packages/react-examples/src/ComponentExamples.test.tsx

```diff
@@ -130,15 +130,7 @@ function getPackageAndExampleName(examplePath: string): [string, string] {
 }
 
 /** Run tests on these packages' examples */
-const includedPackages = [
-  'react',
-  'react-button',
-  'react-checkbox',
-  'react-focus',
-  'react-link',
-  'react-tabs',
```

Comment:
Can you delete packages/react-examples/src/react-tabs/__snapshots__ as well? I guess that package still has examples in react-examples too (I forgot about that) but they're not at all important to test since we won't be using that implementation moving forward.